src/components/Detail/Detail.tsx

```diff
@@ -0,0 +1,99 @@
+import { useRouter } from 'next/router';
+import classnames from 'classnames';
+
+import { Row } from '../Row/Row';
+import { trpc } from '../../utils/trpc';
+import style from './Detail.module.scss';
+
+export const DetailOfActivity = () => {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const utils = trpc.useContext();
+
+  const { data: activityData } = trpc.activities.read.useQuery({
+    id: Number(id),
+  });
+
+  const deleteActivity = trpc.activities.delete.useMutation({
+    onSuccess: () => {
+      utils.activities.invalidate();
+    },
+  });
+
+  const handleDelete = (activityId: number) => {
+    deleteActivity.mutate({ id: activityId });
+
+    router.push('/');
+  };
+
+  const updateDoneCounter = trpc.activities.updateDoneCounter.useMutation({
+    onSuccess: () => {
+      utils.activities.invalidate();
+    },
+  });
+
+  const handleActivityClick = ({
+    activityId,
+    timesDone,
+  }: {
+    activityId: number;
+    timesDone: number;
+  }) => {
+    updateDoneCounter.mutate({ id: activityId, timesDone: timesDone + 1 });
+  };
+
+  const days = (timesDone: number) => {
```

Comment:
Good enough!

Reply:
Also, this would work.

```
  const timeMappings = {
    1: 'den',
    2: 'dny',
    3: 'dny',
    4: 'dny',
  };

 return  timeMappings[timesDone] || 'dní';
```